communication/src/initialize.rs

```diff
@@ -18,7 +18,7 @@ use logging_core::Logger;
 
 
 /// Possible configurations for the communication infrastructure.
-pub enum Configuration {
+pub enum CommunicationConfig {
```

Comment:
I feel like the "recommended style" is leaving this as `Configuration` but then importing it as `CommunicationConfig` in `timely`. I don't think it matters too much outside of this crate (as it will be `CommunicationConfig` by timely users). Any thoughts?